src/main/java/com/gmail/chernii/oleksii/ListXML.java

```diff
@@ -0,0 +1,57 @@
+package com.gmail.chernii.oleksii;
+
+import java.beans.XMLEncoder;
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Created by Space on 25.03.2019.
+ */
+public class ListXML {
+    public static final String FILE_NAME = "list.xml";
+
+    public static void listToXML(List<Map<List<Set<Integer>>, String>> list) {
+        try (BufferedOutputStream bufferedOutputStream = new BufferedOutputStream(new FileOutputStream(FILE_NAME));
+             XMLEncoder xmlEncoder = new XMLEncoder(bufferedOutputStream)) {
+            xmlEncoder.writeObject(list);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static Set<Integer> createSet() {
+        Set<Integer> set = new HashSet<>();
+        set.add(1);
+        set.add(2);
+        set.add(3);
+        return set;
+    }
+
+    private static List<Set<Integer>> createInnerList() {
+        List<Set<Integer>> list = new ArrayList<>();
+        list.add(createSet());
+        list.add(createSet());
+        return list;
+    }
+
+    private static Map<List<Set<Integer>>, String> createMap() {
+        Map<List<Set<Integer>>, String> map = new HashMap<>();
+        map.put(createInnerList(), "value");
+        return map;
+    }
+
+    public static List<Map<List<Set<Integer>>, String>> createList() {
```

Comment:
firstly write public methods, then private
[Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)